src/node/init/init.ts

```diff
@@ -50,6 +51,13 @@ export async function init(root: string | undefined) {
         })
       },
 
+      srcDir: async () => {
+        return text({
+          message: 'Where should VitePress look for your markdown files?',
+          initialValue: './'
```

Comment:
I think this should default to root, and should be resolved relative to cwd. srcDir is resolved relative to root, but I think this is a question and should be answered relative to current context.

this looks weird:

<img width="423" alt="image" src="https://github.com/user-attachments/assets/549d8066-c7fc-4aaa-947d-8468c152524b" />